### PR TITLE
Dev/homepage

### PR DESCRIPTION
**Pull request by loganathan-s**
Hi @smuemd / @gitreco 

I've implemented the homepage and made some changes to existing views, please review and merge it if everything looks good for you.

1. Added authlevel to localstorage, to make the authlevel persistent. Rest of the diffs are due to pretty formatter.

 const localStore = window.localStorage
 let extId = localStore.getItem('user:extId') || undefined
 let token = localStore.getItem('user:token') || undefined
+let authLevel = localStore.getItem('user:authLevel') || undefined
 
```
 export default function createModel (settings) {
   return {
     page: undefined,
-    routesDesc: settings.routesDesc, /* 1 */
+    routesDesc: settings.routesDesc /* 1 */,
     routeName: settings.routesDesc[0].name,
     routeRedirect: false,
     params: {},
     user: {
       extId: extId,
       token: token,
-      authLevel: undefined,
+      authLevel: authLevel,
       localNode: {
         __persistentAddress: undefined,
         __persistentPk: undefined,
-        address: localStore.getItem('user[' + extId + ']:localNode:address') || undefined,
+        address:
+          localStore.getItem('user[' + extId + ']:localNode:address') ||
+          undefined,
         pk: localStore.getItem('user[' + extId + ']:localNode:pk') || undefined
       },
       remoteNode: {
-        address: localStore.getItem('user[' + extId + ']:remoteNode:address') || undefined
+        address:
+          localStore.getItem('user[' + extId + ']:remoteNode:address') ||
+          undefined
       }
     },
     partnerAccount: {
       status: undefined,
-      address: localStore.getItem('user[' + extId + ']:remoteNode:address') || undefined,
+      address:
+        localStore.getItem('user[' + extId + ']:remoteNode:address') ||
+        undefined,
       haben: undefined,
       soll: undefined,
       saldo: undefined,

```
2. Added new views called vwHelper, to have common view helper methods
3. Added new views for Account & Transactions
4. Removed vwItem since we no longer need it

### CODE REVIEW

**Comment by loganathan-s** [comment]
Hi @smuemd 

I'll make sure to run the test before pushing and I'll convert the Header view into smaller ones.

**Comment by smuemd** [comment]
I wouldn't neccicarrily split vwHeader into seperate files.
But maybe using seperate functions for the snippets inside vwHeader/index

However this is not critical ad will probably see some more changes down the road anyway. So we could also save it for then.

**Comment by smuemd** [comment]
take a look at vwAccount and model to seperate the account you are looking at from the 'user data' in the model.

I left that to you because I think understanding that distinction is important.

talk to you in a bit
-S

File: assets/js/view/vwAccount/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    m('div.pa5.mt10', [
+      m(
+        'div',
+        m('dl', { class: 'dib mr4 mt0' }, [
+          m('dt', { class: 'f6 db' }, 'Provisionkonto'),
+          m(
+            'dd',
+            { class: 'db f2 pl0 ml0 f4 b' },
+            model.user && model.user.remoteNode
+              ? model.user.remoteNode.address
```
Comment (smuemd) [inline assets/js/view/vwAccount/index.js]: ☝️ that one line 24

**Pull request merged**